prime-router/metadata/hl7_mapping/ORU_R01/RADx_MARS/RADx_MARS-base.yml

```
@@ -55,4 +55,10 @@ elements:
 
   # OBR-4
   - name: universal-service
-    condition: false
\ No newline at end of file
+    condition: false
+
+  # XTN
+  - name: xtn-use-code-from-system
+    condition: '%resource.system = "email"'
+    value: [ '"NET"' ]
+    hl7Spec: [ '%{hl7TelecomPath}-2' ]
```

This highlights a slight downside to having multiple elements with the same name in referenced schemas; we can't really replicate the behavior of having NET for email and BPN for pager without adding another element with a new name here. This is a good case to keep in mind when/if we do some schema extension/overriding redesign at some point.

We probably are fine without a BPN element here, but we can utilize the override to just barely shorten this, totally optional.

```suggestion
  - name: xtn-use-code-from-system
    condition: '%resource.system = "email"'
    value: [ '"NET"' ]
```

---

Same optional comment applies to the CA transforms.